src/app/main/projects/project-detail/project-detail.component.html

```diff
@@ -0,0 +1,15 @@
+<div class="jumbotron jumbotron-fluid">
+
+  <div class="container">
+    <button
+            (click)="goBack()"><img src="../../../../assets/images/projects/back.png" width="50px" height="50px"></button>
```

Comment:
use absolute path:
`src="./assets/images/projects/back.png"`

place styles in scss file
`width` and `height`